python/tskit/trees.py

```diff
@@ -4202,6 +4212,10 @@ def at(self, position, **kwargs):
     def at_index(self, index, **kwargs):
         """
         Returns the tree at the specified index. See also :meth:`Tree.seek_index`.
+        Note that as tree sequences are stored tree-by-tree this method has to pass
```

Comment:
Did you mean to keep this sentence as well as LinearTraversalWarning?

Reply:
I'm still arguing with sphinx syntax...